.github/workflows/InvalidationFlagger.yml

```
@@ -0,0 +1,33 @@
+name: Invalidations
+on: [push]
```

```suggestion
on: [push, pull_request]
```

so that PRs from another fork also trigger this test.